### PR TITLE
[Update] Facebook for iOS v4.38.0

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

**Comment by SotoiGhost** [comment]
@dalexsoto same process as always heh. Thanks, dude!

* [ ] Facebook Account Kit
* [ ] Facebook Core Kit
* [ ] Facebook Login Kit
* [ ] Facebook Share Kit

File: Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs [hunk 1]
```diff
@@ -86,6 +86,237 @@ interface AdChoicesView {
...
+		[NullAllowed, Export ("contentURL")]
+		string ContentUrl { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable extraData;
+		[NullAllowed, Export ("extraData")]
+		string ExtraData { get; set; }
+
+		// -(instancetype _Nonnull)initWithKeywords:(NSArray<FBAdExtraHintKeyword> * _Nonnull)keywords;
+		[Export ("initWithKeywords:")]
+		IntPtr Constructor (string [] keywords);
```
Comment (dalexsoto) [inline Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs]: This is a `NSArray<FBAdExtraHintKeyword>` so it needs keys from `AdExtraHintKeywords`

```suggestion
		IntPtr Constructor (NSString [] keywords);
```

File: Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs [hunk 2]
```diff
@@ -86,6 +86,237 @@ interface AdChoicesView {
...
+
+		// -(void)addKeyword:(FBAdExtraHintKeyword _Nonnull)keyword;
+		[Export ("addKeyword:")]
+		void AddKeyword (string keyword);
```
Comment (dalexsoto) [inline Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs]: needs  `FBAdExtraHintKeyword`

```suggestion
		void AddKeyword (NSString keyword);
```

File: Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs [hunk 3]
```diff
@@ -86,6 +86,237 @@ interface AdChoicesView {
...
+
+		// -(void)removeKeyword:(FBAdExtraHintKeyword _Nonnull)keyword;
+		[Export ("removeKeyword:")]
+		void RemoveKeyword (string keyword);
```
Comment (dalexsoto) [inline Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs]: needs `FBAdExtraHintKeyword`

```suggestion
		void RemoveKeyword (NSString keyword);
```

File: Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs (hunk 1 above)
Comment (SotoiGhost) [reply Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs]: You have a point, good sir! Changing it using beta tool.

**Comment by newky2k** [comment]
fixed #92 #94 #89

**Pull request merged**